cmp/project.py

```diff
@@ -394,40 +391,46 @@ def init_dmri_project(project_info, bids_layout, is_new_project, gui=True, debug
             dmri_pipeline.input_folders,
             session=project_info.subject_session,
         )
+        subject_derivatives_directory = os.path.join(
+            derivatives_directory, project_info.subject, project_info.subject_session
+        )
     else:
         refresh_folder(
             bids_directory,
             derivatives_directory,
             project_info.subject,
             dmri_pipeline.input_folders,
         )
+        subject_derivatives_directory = os.path.join(
+            derivatives_directory, project_info.subject
+        )
 
     dmri_inputs_checked = dmri_pipeline.check_input(layout=bids_layout, gui=gui)
     if dmri_inputs_checked:
         if (
             is_new_project and dmri_pipeline is not None
-        ):  # and dmri_pipelineis not None:
+        ):
             print("> Initialize dmri project")
-            if not os.path.exists(derivatives_directory):
+            if not os.path.exists(subject_derivatives_directory):
                 try:
-                    os.makedirs(derivatives_directory)
+                    os.makedirs(subject_derivatives_directory)
                 except os.error:
-                    print("... Info : %s was already existing" % derivatives_directory)
+                    print("... Info : %s was already existing" % subject_derivatives_directory)
                 finally:
-                    print("... Info : Created directory %s" % derivatives_directory)
+                    print("... Info : Created directory %s" % subject_derivatives_directory)
```

Comment:
This print statement ("Created directory") should be moved inside the try block because finally gets evaluated regardless of the success of the try block.